build.gradle

```diff
@@ -157,10 +134,11 @@ dependencies {
   api group: 'org.springframework.boot', name: 'spring-boot-starter-actuator'
   api group: 'io.github.openfeign', name: 'feign-jackson'
   annotationProcessor group: 'org.projectlombok', name: 'lombok', version: '1.18.6'
-  compileOnly group: 'org.projectlombok', name: 'lombok', version: '1.18.6'
-  testCompile group: 'junit', name: 'junit', version: '4.12'
-  testCompile group: 'org.mockito', name: 'mockito-core', version: '2.23.0'
-
+  implementation group: 'org.projectlombok', name: 'lombok', version: '1.18.6'
```

Comment:
lombok should not be a dependency of the project I think `compileOnly` is still valid:
https://projectlombok.org/setup/gradle